test/t8_IO/t8_gtest_vtk_writer.cxx

```diff
@@ -91,6 +129,12 @@ class vtk_writer_test: public testing::Test {
                                      sc_MPI_COMM_WORLD);
   }
 
+  int
```

Comment:
```suggestion
static int
```

Reply:
This doesn't work, because the function uses grid, which is a non-static member variable. Static member functions can't use non-static member variables.